pom.xml

```diff
@@ -184,6 +187,24 @@
         <version>${project.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.arquillian.cube</groupId>
+        <artifactId>arquillian-cube-openshift-starter</artifactId>
+        <version>${project.version}</version>
```

Comment:
why we don't have `<scope>test</scope>`? In case if user forget to override scope while using this dependency in submodule.